src/migration/PoC_dfusion.js

```diff
@@ -27,28 +27,25 @@ async function migrate({ artifacts, deployer, network, account, web3, maxTokens
     account,
     "@gnosis.pm/dex-contracts/build/contracts/BatchExchange"
   )
-  const BiMap = getDependency(
-    artifacts,
-    network,
-    deployer,
-    account,
-    "@gnosis.pm/solidity-data-structures/build/contracts/IdToAddressBiMap"
-  )
-  const IterableAppendOnlySet = getDependency(
-    artifacts,
-    network,
+  const BiMap = artifactFromNpmImport("@gnosis.pm/solidity-data-structures/build/contracts/IdToAddressBiMap", deployer, account)
+  const biMap = await BiMap.deployed()
+  biMap.contract_name = "IdToAddressBiMap"
```

Comment:
These field contract_name and isDeployed are expected, but they are not set by @truffle/contract